src/RulesTest/MonsterTest.cs

```diff
@@ -102,7 +102,7 @@ public void MonsterButtonTest()
             Assert.AreEqual(EvaluationCode.NotApplicable, results[RuleId.IsContentElementTrueOptional]);
             Assert.AreEqual(EvaluationCode.Pass, results[RuleId.IsControlElementPropertyExists]);
             Assert.AreEqual(EvaluationCode.NotApplicable, results[RuleId.IsControlElementTrueOptional]);
-            Assert.AreEqual(EvaluationCode.Pass, results[RuleId.IsControlElementTrueRequired]);
+            Assert.AreEqual(EvaluationCode.NotApplicable, results[RuleId.IsControlElementTrueRequired]);
```

Comment:
@DaveTryon Would you mind sharing a little bit about this change?

Reply:
Since this is a WPF button, the new rule applies instead of the old one. I'll add a line to capture the new rule